azerothcore/azerothcore-wotlk
# fix(DB/creature_formations): Update Hellfire Ramparts GroupAI and link stationary packs

## Changes Proposed:
-  Links all stationary packs
-  Makes boss' adds respawn on reset

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Tested ingame

## How to Test the Changes:


1. attack a creature, see if the pack aggroes immediately
2. kill them all, see if no more are aggroed
3. kill the 1st boss adds, see if they respawn once you .combatstop

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
